public/articles-publications.json

```diff
@@ -1,129 +1,129 @@
 {
   "articles": [
     {
-      "title": "THE FIGHT TO SAVE THE WORLD’S SEAS FROM CHINA’S BLOATED FISHING INDUSTRY",
+      "title": "The fight to save the world’s seas from China’s bloated fishing industry",
       "link": "http://time.com/4463943/japan-china-fishing-marine-iuu-environment-google-skytruth/",
       "author": "Charlie Campbell, Time Magazine",
       "date": "August 24, 2016"
     },
     {
-      "title": "IS THAT REAL TUNA IN YOUR SUSHI? NOW, A WAY TO TRACK THAT FISH",
+      "title": "Is that real tuna in your sushi? Now, a way to track that fish",
       "link": "http://www.nytimes.com/2016/08/14/technology/is-that-real-tuna-in-your-sushi-now-a-way-to-track-that-fish.html?_r=1",
       "author": "Claire Martin, The New York Times",
       "date": "August 13, 2016"
     },
     {
-      "title": "THINK YOU BOUGHT RED SNAPPER? DON’T BE SO SURE",
+      "title": "Think you bought red snapper? Don’t be so sure",
       "link": "http://www.wsj.com/articles/think-you-bought-red-snapper-dont-be-so-sure-1470428804",
       "author": "Larry Olmstead, The Wall Street Journal",
       "date": "August 5, 2016"
     },
     {
-      "title": "MOST SUSHI IN THE US ISN'T WHAT YOU THINK IT IS AND NEW TECHNOLOGY COULD HELP FIX THAT",
+      "title": "Most sushi in the US isn't what you think it is and new technology could help fix that",
       "link": "http://www.businessinsider.com/most-sushi-isnt-what-you-think-it-is-2016-8",
       "author": "Kate Taylor",
       "date": "August 15, 2016"
     },
     {
-      "title": "PALAU VS. THE POACHERS",
+      "title": "Palau vs. the Poachers",
       "link": "http://www.nytimes.com/2016/02/21/magazine/palau-vs-the-poachers.html",
```

Comment:
Maybe is "The Poachers".
@clementprdhomme 
Is more like a question, I not sure.

Ready to merge if is with "the" and not "The"